IntegrationTests/AsyncStorageTest.js

```diff
@@ -71,7 +71,8 @@ function testSetAndGet() {
     AsyncStorage.getItem(KEY_1, (err2, result) => {
       expectAsyncNoError('testSetAndGet/getItem', err2);
       expectEqual(result, VAL_1, 'testSetAndGet setItem');
-      updateMessage('get(key_1) correctly returned ' + result);
+      expectTrue(result !== null, 'should not be null');
```

Comment:
Aren't these expectations covered by the previous statement? `VAL_1` isn't null, so we know that result isn't null.